tests/integration/alb/test_alb_provisioning.py

```diff
@@ -69,103 +68,6 @@ def test_gets_lowest_used_alb(alb):
     ) == ("alb-listener-arn-1", "listener-arn-1")
 
 
-def test_refuses_to_provision_synchronously(client):
```

Comment:
Why are we deleting these tests?

Reply:
these tests were identical except for the plan type we were calling, so I merged them all into `common_checks.py`, where we test all of them using `@pytest.mark.parametrize` (except I missed migration instances, as you pointed out in another comment - I'll fix that)